cli/cmdlineparser.cpp

```diff
@@ -1688,6 +1695,7 @@ void CmdLineParser::printHelp() const
         "    --file-list=<file>   Specify the files to check in a text file. Add one\n"
         "                         filename per line. When file is '-,' the file list will\n"
         "                         be read from standard input.\n"
+        "    --filesdir           Print the built-in FILESDIR.\n"
```

Comment:
hmm.. nobody has asked for this as far as I know. why do you add it?

Reply:
The interface of some linux tools is not very nice. Lots of options. I don't want to end up with 100's of options.

Reply:
> hmm.. nobody has asked for this as far as I know.

Most of the issue we uncover have not been reported by the users. Would that be a reason not to fix them.

> why do you add it?

Because outside of some debug messages (and that only recently) that path is not visible unless you dump all the strings used in the executable.

Also it is an absolute(!) path hard-coded by the packager(!) which a normal user cannot control.

And see the ticket for more reasoning.